tools/codegen/config.yml

```diff
@@ -444,3 +444,21 @@ resources:
           computability:
             optional: true
             computed: true
+
+  org_service_account_api:
+    read:
+      path: /api/atlas/v2/orgs/{orgId}/serviceAccounts/{clientId}
+      method: GET
+    create:
+      path: /api/atlas/v2/orgs/{orgId}/serviceAccounts
+      method: POST
+    update:
+      path: /api/atlas/v2/orgs/{orgId}/serviceAccounts/{clientId}
+      method: PATCH
+    delete:
+      path: /api/atlas/v2/orgs/{orgId}/serviceAccounts/{clientId}
+      method: DELETE
+    schema:
+      overrides:
+        secrets.secret:
+          sensitive: true
```

Comment:
currently `secret` property does not have `format: password` defined in the API Spec, defined override but also captured in CLOUDP-348768.